webui/react/src/shared/bin/test.py

```diff
@@ -80,7 +84,7 @@ def run_webui_tests(web_dir: pathlib.Path):
     """
     runs webui tests. requires web_dir to be set up
     """
-    for target in ['get-deps', 'check', 'build', 'test']:
+    for target in ['check', 'build', 'test']:
```

Comment:
👍

Reply:
mainy needed because it triggers a submodule update which can unwind previous changes from the `*local*` function